### DISCUSSION TITLE
How to set flutter version for linux package

### DISCUSSION THREAD

**Discussion by cuishijie1991** [codemagic-ci-cd/codemagic-docs#1662] [Q&A] [answered]
Hello, we would like to use your company's products for remote packaging and release on linux Ubuntu. 
But we found that the default version of flutter was the latest version when we packaged it, and our app could not use the latest version of flutter due to plugin compatibility. 
Do you know how to configure it

**Comment by dtrdic** [comment, marked as answer]
hey @cuishijie1991 there is a new feature in codemagic that lets you use Flutter version of your project, you will need to install Flutter Version Management (FVM). Check out more about this here https://docs.codemagic.io/yaml-quick-start/building-a-flutter-app/#setting-the-flutter-version
